README.md

```diff
@@ -6,30 +6,26 @@ The aim of this project is to show how `fastify` can be used.
 
 ## Run
 
-### Pseudo-production
+For running this project on your machine:
 ```bash
-# build frontend
-cd frontend && npm run build && cd ..
-# start backend
-npm start -- --log-level trace --port 3001
-```
-
-Open your browser at [http://localhost:3001](http://localhost:3001)
-
-### Development
-```bash
-# start frontend server
+# terminal 1: Start frontend server
 cd frontend && npm start
-# start backend
-npm start -- --log-level trace --port 3001 # in another terminal
+# terminal 2: start backend
+set -a
+source local.env
+npm start -- --log-level trace --port 3001
```

Comment:
you might want to use https://github.com/toddbluhm/env-cmd or https://www.npmjs.com/package/cross-env

Reply:
You might also consider using https://www.npmjs.com/package/nixconfig and https://www.npmjs.com/package/nixconfig-yaml. They eliminate needing to worry about where the configuration comes from. An example of configuration that uses it -- https://github.com/jscas/cas-server/blob/fastify/lib/config.js

Reply:
Done